sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -597,6 +575,14 @@ private static DefaultHostVolumeSpec convertHostVolume(RawHostVolume rawHostVolu
         .build();
   }
 
+  private static ExternalVolumeSpec convertHostVolume(RawExternalVolume rawExternalVolume) {
```

Comment:
```suggestion
  private static ExternalVolumeSpec convertExternalVolume(RawExternalVolume rawExternalVolume) {
```

I think you meant `convertExternalVolume` here instead of `convertHostVolume`